src/release/breaking-changes/3-16-deprecations.md

```diff
@@ -0,0 +1,276 @@
+---
+title: Deprecated API removed after v3.16
+description: >-
+After reaching end of life, the following deprecated APIs
+were removed from Flutter.
```

Comment:
I think this is breaking the build since YAML expects indentation here:

```suggestion
  After reaching end of life, the following deprecated APIs
  were removed from Flutter.
```

Reply:
Ah thank you! A bit of copy pasta. 🍝 
I feel like I always hit this when I write these because the indents don't follow.